esmvaltool/utils/recipe_test_workflow/recipe_test_workflow/app/configure/bin/configure.py

```diff
@@ -17,16 +14,11 @@ def main():
     environment for the ``configure`` task and the result is written to
     the file defined by ``USER_CONFIG_PATH`` in the ``flow.cylc`` file.
```

Comment:
Would it be possible for you to update this docstring to remove the reference to "default configuration values from the latest version of ESMValTool", please? :)

Reply:
Thanks for catching this! https://github.com/ESMValGroup/ESMValTool/pull/3394/commits/33b08020f94dff660a809a9ff5818dd0d370b3d5